### ISSUE TITLE
Add a clock (using the RTC)

### ISSUE THREAD

**Issue by boricj** [numworks/epsilon#139]
The firmware does not use it currently, but the SFM32F412 chip has a RTC with an internal oscillator source available. The calculator ought to be able to tell the time to the user.

**Comment by Ecco** [comment]
The problem with using the RTC is its calibration. We haven't benchmarked the RTC but it might end up drifting quite a bit which would end up a bit counter-productive.

That being said, and even if the RTC were to be drifting, it could still be used to show the remaining time during an exam. Indeed, on such a small timeframe (a few hours) drifting will be completely negligible. And that's the most important time-related information one may want on a calculator.

**Comment by boricj** [comment]
Yes, even if it drifts tens of seconds per day it's still good enough to serve as a time reference for students during exams.

According to the datasheet the RTC can do smooth digital calibration with a ~30s cycle, so if the internal RTC oscillator is that bad it should be possible to calibrate it on-the-fly when the calc is powered on.

**Comment by RedGl0w** [comment]
Note : a PR has been opened on omega for RTC https://github.com/Omega-Numworks/Omega/pull/347

**Comment by boricj** [comment]
While it works, I highly doubt that NumWorks will accept this into epsilon unless a new hardware revision with a LSE oscillator is released. The other two options for clocking the RTC are either far too inaccurate (LSI) or extremely power-hungry (HSE).
